src/Controller/Component/PaginatorComponent.php

```diff
@@ -34,15 +34,15 @@
  * You configure pagination when calling paginate(). See that method for more details.
  *
  * @link https://book.cakephp.org/4/en/controllers/components/pagination.html
- * @mixin \Cake\Datasource\Paging\Paginator
+ * @mixin \Cake\Datasource\Paging\DefaultPaginator
```

Comment:
Are you open to calling this `BasicPaginator` instead? We used `Default` for password hashing in Authentication and that name adds a layer of indirection to finding out what the 'default' is. With 'basic' we can infer a bit more than we can with 'default'.

Reply:
I am fine with renaming the class. But we already have a `SimplePaginator`, so `BasicPaginator` becomes too similar. Can we name the default class as `NumericPaginator` or `SlidingPaginator`? I am open to other suggestions too as long as we can easily distinguish the 2 paginators we have.

Reply:
🤦  I forgot we already had `SimplePaginator`. Totally agree that having both Simple and Basic is 🙃 . `NumericPaginator` works for me. It still leaves room for cursor and key based paginators in the future.

Reply:
Renamed `DefaultPaginator` to `NumericPaginator`.